extensions-contrib/scan-query/src/main/java/io/druid/query/scan/ScanResultValue.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package io.druid.query.scan;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+public class ScanResultValue implements Comparable<ScanResultValue>
+{
+  public static final String timestampKey = "timestamp";
```

Comment:
in other places, we use `__time` e.g. in groupBy queries etc. It might make sense to be consistent. Also, you can probably refer to Column.TIME_COLUMN_NAME

Reply:
@himanshug why you don't used "__time" instead of "timestamp"